hack/e2e-test.sh

```diff
@@ -61,4 +63,4 @@ trap cleanup EXIT
 startup
 kind_load
 kueue_deploy
-$GINKGO --junit-report=junit.xml --output-dir=$ARTIFACTS -v ./test/e2e/...
+$GINKGO --junit-report=junit.xml --output-dir=$ARTIFACTS -v ./test/e2e/... || (kubectl get nodes; kubectl get pods -n kube-system; kubectl get pods -n kueue-system)
```

Comment:
just kueue-system should be enough, and we can maybe use describe, as it gives more information.

I would also consider sending the output to a file instead.
See the function cleanup.

Reply:
I will do that.